### ISSUE TITLE
2025-05-30, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#185]
# Today's Arxiv Papers

(1) [Argus: Vision-Centric Reasoning with Grounded Chain-of-Thought](http://arxiv.org/abs/2505.23766v1), 2025-05-29

(2) [From Chat Logs to Collective Insights: Aggregative Question Answering](http://arxiv.org/abs/2505.23765v1), 2025-05-29

(3) [MMSI-Bench: A Benchmark for Multi-Image Spatial Intelligence](http://arxiv.org/abs/2505.23764v1), 2025-05-29

(4) [DeepTheorem: Advancing LLM Reasoning for Theorem Proving Through Natural Language and Reinforcement Learning](http://arxiv.org/abs/2505.23754v1), 2025-05-29

(5) [ThinkGeo: Evaluating Tool-Augmented Agents for Remote Sensing Tasks](http://arxiv.org/abs/2505.23752v1), 2025-05-29

(6) [Distortion of AI Alignment: Does Preference Optimization Optimize for Preferences?](http://arxiv.org/abs/2505.23749v1), 2025-05-29

(7) [Spatial-MLLM: Boosting MLLM Capabilities in Visual-based Spatial Intelligence](http://arxiv.org/abs/2505.23747v1), 2025-05-29

(8) [Bounded Rationality for LLMs: Satisficing Alignment at Inference-Time](http://arxiv.org/abs/2505.23729v1), 2025-05-29

(9) [PixelThink: Towards Efficient Chain-of-Pixel Reasoning](http://arxiv.org/abs/2505.23727v1), 2025-05-29

(10) [MuLoCo: Muon is a practical inner optimizer for DiLoCo](http://arxiv.org/abs/2505.23725v1), 2025-05-29

(11) [SC-LoRA: Balancing Efficient Fine-tuning and Knowledge Preservation via Subspace-Constrained LoRA](http://arxiv.org/abs/2505.23724v1), 2025-05-29

(12) [ML-Agent: Reinforcing LLM Agents for Autonomous Machine Learning Engineering](http://arxiv.org/abs/2505.23723v1), 2025-05-29

(13) [Label-Guided In-Context Learning for Named Entity Recognition](http://arxiv.org/abs/2505.23722v1), 2025-05-29

(14) [TiRex: Zero-Shot Forecasting Across Long and Short Horizons with Enhanced In-Context Learning](http://arxiv.org/abs/2505.23719v1), 2025-05-29

(15) [Don't Take the Premise for Granted: Evaluating the Premise Critique Ability of Large Language Models](http://arxiv.org/abs/2505.23715v1), 2025-05-29

(16) [SocialMaze: A Benchmark for Evaluating Social Reasoning in Large Language Models](http://arxiv.org/abs/2505.23713v1), 2025-05-29

(17) [Let's Reason Formally: Natural-Formal Hybrid Reasoning Enhances LLM's Math Capability](http://arxiv.org/abs/2505.23703v1), 2025-05-29

(18) [Can LLMs Reason Abstractly Over Math Word Problems Without CoT? Disentangling Abstract Formulation From Arithmetic Computation](http://arxiv.org/abs/2505.23701v1), 2025-05-29

(19) [Data-to-Dashboard: Multi-Agent LLM Framework for Insightful Visualization in Enterprise Analytics](http://arxiv.org/abs/2505.23695v1), 2025-05-29

(20) [VF-Eval: Evaluating Multimodal LLMs for Generating Feedback on AIGC Videos](http://arxiv.org/abs/2505.23693v1), 2025-05-29

(21) [Fortune: Formula-Driven Reinforcement Learning for Symbolic Table Reasoning in Language Models](http://arxiv.org/abs/2505.23667v1), 2025-05-29

(22) [LoLA: Low-Rank Linear Attention With Sparse Caching](http://arxiv.org/abs/2505.23666v1), 2025-05-29

(23) [ToolHaystack: Stress-Testing Tool-Augmented Language Models in Realistic Long-Term Interactions](http://arxiv.org/abs/2505.23662v1), 2025-05-29

(24) [OpenUni: A Simple Baseline for Unified Multimodal Understanding and Generation](http://arxiv.org/abs/2505.23661v1), 2025-05-29

(25) [D-AR: Diffusion via Autoregressive Models](http://arxiv.org/abs/2505.23660v1), 2025-05-29

(26) [Active Layer-Contrastive Decoding Reduces Hallucination in Large Language Model Generation](http://arxiv.org/abs/2505.23657v1), 2025-05-29

(27) [ARC: Argument Representation and Coverage Analysis for Zero-Shot Long Document Summarization with Instruction Following LLMs](http://arxiv.org/abs/2505.23654v1), 2025-05-29

(28) [How does Transformer Learn Implicit Reasoning?](http://arxiv.org/abs/2505.23653v1), 2025-05-29

(29) [MCP Safety Training: Learning to Refuse Falsely Benign MCP Exploits using Improved Preference Alignment](http://arxiv.org/abs/2505.23634v1), 2025-05-29

(30) [AutoSchemaKG: Autonomous Knowledge Graph Construction through Dynamic Schema Induction from Web-Scale Corpora](http://arxiv.org/abs/2505.23628v1), 2025-05-29

(31) [Table-R1: Inference-Time Scaling for Table Reasoning](http://arxiv.org/abs/2505.23621v1), 2025-05-29

(32) [A Comprehensive Evaluation of Multi-Modal Large Language Models for Endoscopy Analysis](http://arxiv.org/abs/2505.23601v1), 2025-05-29

(33) [LLM Performance for Code Generation on Noisy Tasks](http://arxiv.org/abs/2505.23598v1), 2025-05-29

(34) [MAPLE: A Mobile Assistant with Persistent Finite State Machines for Recovery Reasoning](http://arxiv.org/abs/2505.23596v1), 2025-05-29

(35) [Jigsaw-R1: A Study of Rule-based Visual Reinforcement Learning with Jigsaw Puzzles](http://arxiv.org/abs/2505.23590v1), 2025-05-29

(36) [On-Policy RL with Optimal Reward Baseline](http://arxiv.org/abs/2505.23585v1), 2025-05-29

(37) [Engineering Serendipity through Recommendations of Items with Atypical Aspects](http://arxiv.org/abs/2505.23580v1), 2025-05-29

(38) [BioReason: Incentivizing Multimodal Biological Reasoning within a DNA-LLM Model](http://arxiv.org/abs/2505.23579v1), 2025-05-29

(39) [Cognitive Guardrails for Open-World Decision Making in Autonomous Drone Swarms](http://arxiv.org/abs/2505.23576v1), 2025-05-29

(40) [Evaluating AI capabilities in detecting conspiracy theories on YouTube](http://arxiv.org/abs/2505.23570v1), 2025-05-29

(41) [Segment Policy Optimization: Effective Segment-Level Credit Assignment in RL for Large Language Models](http://arxiv.org/abs/2505.23564v1), 2025-05-29

(42) [Merge Hijacking: Backdoor Attacks to Model Merging of Large Language Models](http://arxiv.org/abs/2505.23561v1), 2025-05-29

(43) [SafeScientist: Toward Risk-Aware Scientific Discoveries by LLM Agents](http://arxiv.org/abs/2505.23559v1), 2025-05-29

(44) [Understanding Refusal in Language Models with Sparse Autoencoders](http://arxiv.org/abs/2505.23556v1), 2025-05-29

(45) [Adaptive Federated LoRA in Heterogeneous Wireless Networks with Independent Sampling](http://arxiv.org/abs/2505.23555v1), 2025-05-29

(46) [Sustainable Carbon-Aware and Water-Efficient LLM Scheduling in Geo-Distributed Cloud Datacenters](http://arxiv.org/abs/2505.23554v1), 2025-05-29

(47) [LLM-based Property-based Test Generation for Guardrailing Cyber-Physical Systems](http://arxiv.org/abs/2505.23549v1), 2025-05-29

(48) [Translation in the Wild](http://arxiv.org/abs/2505.23548v1), 2025-05-29

(49) [Position Paper: Metadata Enrichment Model: Integrating Neural Networks and Semantic Knowledge Graphs for Cultural Heritage Applications](http://arxiv.org/abs/2505.23543v1), 2025-05-29

(50) [Probability-Consistent Preference Optimization for Enhanced LLM Reasoning](http://arxiv.org/abs/2505.23540v1), 2025-05-29

(51) [Domain-Aware Tensor Network Structure Search](http://arxiv.org/abs/2505.23537v1), 2025-05-29

(52) [AnchorAttention: Difference-Aware Sparse Attention with Stripe Granularity](http://arxiv.org/abs/2505.23520v1), 2025-05-29

(53) [VAU-R1: Advancing Video Anomaly Understanding via Reinforcement Fine-Tuning](http://arxiv.org/abs/2505.23504v1), 2025-05-29

(54) [Can Large Language Models Challenge CNNS in Medical Image Analysis?](http://arxiv.org/abs/2505.23503v1), 2025-05-29

(55) [Identity resolution of software metadata using Large Language Models](http://arxiv.org/abs/2505.23500v1), 2025-05-29

(56) [Diagnosing and Addressing Pitfalls in KG-RAG Datasets: Toward More Reliable Benchmarking](http://arxiv.org/abs/2505.23495v1), 2025-05-29

(57) [Autoformalization in the Era of Large Language Models: A Survey](http://arxiv.org/abs/2505.23486v1), 2025-05-29

(58) [VCapsBench: A Large-scale Fine-grained Benchmark for Video Caption Quality Evaluation](http://arxiv.org/abs/2505.23484v1), 2025-05-29

(59) [Revisiting Overthinking in Long Chain-of-Thought from the Perspective of Self-Doubt](http://arxiv.org/abs/2505.23480v1), 2025-05-29

(60) [Evaluating the performance and fragility of large language models on the self-assessment for neurological surgeons](http://arxiv.org/abs/2505.23477v1), 2025-05-29

(61) [Socratic-PRMBench: Benchmarking Process Reward Models with Systematic Reasoning Patterns](http://arxiv.org/abs/2505.23474v1), 2025-05-29

(62) [EVOREFUSE: Evolutionary Prompt Optimization for Evaluation and Mitigation of LLM Over-Refusal to Pseudo-Malicious Instructions](http://arxiv.org/abs/2505.23473v1), 2025-05-29

(63) [Synthesizing Performance Constraints for Evaluating and Improving Code Efficiency](http://arxiv.org/abs/2505.23471v1), 2025-05-29

(64) [UAQFact: Evaluating Factual Knowledge Utilization of LLMs on Unanswerable Questions](http://arxiv.org/abs/2505.23461v1), 2025-05-29

(65) [What About Emotions? Guiding Fine-Grained Emotion Extraction from Mobile App Reviews](http://arxiv.org/abs/2505.23452v1), 2025-05-29

(66) [CMIE: Combining MLLM Insights with External Evidence for Explainable Out-of-Context Misinformation Detection](http://arxiv.org/abs/2505.23449v1), 2025-05-29

(67) [Diversity-Aware Policy Optimization for Large Language Model Reasoning](http://arxiv.org/abs/2505.23433v1), 2025-05-29

(68) [SWE-bench Goes Live!](http://arxiv.org/abs/2505.23419v1), 2025-05-29

(69) [KVzip: Query-Agnostic KV Cache Compression with Context Reconstruction](http://arxiv.org/abs/2505.23416v1), 2025-05-29

(70) [From Parameters to Prompts: Understanding and Mitigating the Factuality Gap between Fine-Tuned LLMs](http://arxiv.org/abs/2505.23410v1), 2025-05-29

(71) [A Practical Guide for Supporting Formative Assessment and Feedback Using Generative AI](http://arxiv.org/abs/2505.23405v1), 2025-05-29

(72) [Adaptive Jailbreaking Strategies Based on the Semantic Understanding Capabilities of Large Language Models](http://arxiv.org/abs/2505.23404v1), 2025-05-29

(73) [A Unified Framework for Human AI Collaboration in Security Operations Centers with Trusted Autonomy](http://arxiv.org/abs/2505.23397v1), 2025-05-29

(74) [Afterburner: Reinforcement Learning Facilitates Self-Improving Code Efficiency Optimization](http://arxiv.org/abs/2505.23387v1), 2025-05-29

(75) [UniRL: Self-Improving Unified Multimodal Models via Supervised and Reinforcement Learning](http://arxiv.org/abs/2505.23380v1), 2025-05-29

(76) [Threading the Needle: Reweaving Chain-of-Thought Reasoning to Explain Human Label Variation](http://arxiv.org/abs/2505.23368v1), 2025-05-29

(77) [Discriminative Policy Optimization for Token-Level Reward Models](http://arxiv.org/abs/2505.23363v1), 2025-05-29

(78) [VideoReasonBench: Can MLLMs Perform Vision-Centric Complex Video Reasoning?](http://arxiv.org/abs/2505.23359v1), 2025-05-29

(79) [Understanding the Information Propagation Effects of Communication Topologies in LLM-based Multi-Agent Systems](http://arxiv.org/abs/2505.23352v1), 2025-05-29

(80) [Towards Reward Fairness in RLHF: From a Resource Allocation Perspective](http://arxiv.org/abs/2505.23349v1), 2025-05-29

(81) [Neither Stochastic Parroting nor AGI: LLMs Solve Tasks through Context-Directed Extrapolation from Training Data Priors](http://arxiv.org/abs/2505.23323v1), 2025-05-29

(82) [Proximalized Preference Optimization for Diverse Feedback Types: A Decomposed Perspective on DPO](http://arxiv.org/abs/2505.23316v1), 2025-05-29

(83) [Towards LLM-based Generation of Human-Readable Proofs in Polynomial Formal Verification](http://arxiv.org/abs/2505.23311v1), 2025-05-29

(84) [Generalized Category Discovery in Event-Centric Contexts: Latent Pattern Mining with LLMs](http://arxiv.org/abs/2505.23304v1), 2025-05-29

(85) [Data-efficient Meta-models for Evaluation of Context-based Questions and Answers in LLMs](http://arxiv.org/abs/2505.23299v1), 2025-05-29

(86) [EmoBench-UA: A Benchmark Dataset for Emotion Detection in Ukrainian](http://arxiv.org/abs/2505.23297v1), 2025-05-29

(87) [How Does Response Length Affect Long-Form Factuality](http://arxiv.org/abs/2505.23295v1), 2025-05-29

(88) [ScEdit: Script-based Assessment of Knowledge Editing](http://arxiv.org/abs/2505.23291v1), 2025-05-29

(89) [MathArena: Evaluating LLMs on Uncontaminated Math Competitions](http://arxiv.org/abs/2505.23281v1), 2025-05-29

(90) [Sentinel: Attention Probing of Proxy Models for LLM Context Compression with an Understanding Perspective](http://arxiv.org/abs/2505.23277v1), 2025-05-29

(91) [The Arabic AI Fingerprint: Stylometric Analysis and Detection of Large Language Models Text](http://arxiv.org/abs/2505.23276v1), 2025-05-29

(92) [Wireless Agentic AI with Retrieval-Augmented Multimodal Semantic Perception](http://arxiv.org/abs/2505.23275v1), 2025-05-29

(93) [Does Machine Unlearning Truly Remove Model Knowledge? A Framework for Auditing Unlearning in LLMs](http://arxiv.org/abs/2505.23270v1), 2025-05-29

(94) [Disrupting Vision-Language Model-Driven Navigation Services via Adversarial Object Fusion](http://arxiv.org/abs/2505.23266v1), 2025-05-29

(95) [Image Aesthetic Reasoning: A New Benchmark for Medical Image Screening with MLLMs](http://arxiv.org/abs/2505.23265v1), 2025-05-29

(96) [Can Large Language Models Trigger a Paradigm Shift in Travel Behavior Modeling? Experiences with Modeling Travel Satisfaction](http://arxiv.org/abs/2505.23262v1), 2025-05-29

(97) [MemAscend: System Memory Optimization for SSD-Offloaded LLM Fine-Tuning](http://arxiv.org/abs/2505.23254v1), 2025-05-29

(98) [Deep Retrieval at CheckThat! 2025: Identifying Scientific Papers from Implicit Social Media Mentions via Hybrid Retrieval and Re-Ranking](http://arxiv.org/abs/2505.23250v1), 2025-05-29

(99) [Context-Aware Semantic Communication for the Wireless Networks](http://arxiv.org/abs/2505.23249v1), 2025-05-29

(100) [Accelerating RLHF Training with Reward Variance Increase](http://arxiv.org/abs/2505.23247v1), 2025-05-29
